src/parser.rs

```diff
@@ -624,8 +625,15 @@ impl<'parser> ArgumentParser<'parser> {
 
     /// Create an empty argument parser
     pub fn new() -> ArgumentParser<'parser> {
+        let args: Vec<String> = env::args().collect();
```

Comment:
Unfortunately, you can't do `env::args()` here. Argument parser is often used to parse arguments of subcommands or something like that. I.e. you pass the actual arguments to `parse()` rather than using `env::args()`. You can make "title" an `Option`, and use default in the `parse()` method if it was not overriden by `set_title()`